spark/datadog_checks/spark/spark.py

```diff
@@ -63,92 +63,90 @@
 SOURCE_TYPE_NAME = 'spark'
 
 # Metric types
-INCREMENT = 'increment'
 GAUGE = 'gauge'
 COUNT = 'count'
-MONOTONIC_COUNT = 'monotonic_count'
 
 # Metrics to collect
 SPARK_JOB_METRICS = {
-    'numTasks': ('spark.job.num_tasks', INCREMENT),
-    'numActiveTasks': ('spark.job.num_active_tasks', INCREMENT),
-    'numCompletedTasks': ('spark.job.num_completed_tasks', INCREMENT),
-    'numSkippedTasks': ('spark.job.num_skipped_tasks', INCREMENT),
-    'numFailedTasks': ('spark.job.num_failed_tasks', INCREMENT),
-    'numActiveStages': ('spark.job.num_active_stages', INCREMENT),
-    'numCompletedStages': ('spark.job.num_completed_stages', INCREMENT),
-    'numSkippedStages': ('spark.job.num_skipped_stages', INCREMENT),
-    'numFailedStages': ('spark.job.num_failed_stages', INCREMENT)
+    'numTasks': ('spark.job.num_tasks', COUNT),
```

Comment:
Shouldn't those be `MONOTONIC_COUNT` to get the same value as before?
I think `COUNT` will keep increasing while `MONOTONIC_COUNT` will get us the delta.
We should tests this in a lab in all cases.